packages/framework/src/Resources/config/services.yml

```diff
@@ -973,11 +977,10 @@ services:
 
     Shopsys\FrameworkBundle\Component\Redis\RedisFacade:
         arguments:
-            $cacheClients:
-            - '@snc_redis.bestselling_products'
-            - '@snc_redis.doctrine_metadata'
-            - '@snc_redis.doctrine_query'
-            - '@snc_redis.framework_annotations'
+            $allClients: !tagged snc_redis.client
```

Comment:
will this change be part of upgrade notes ?

Reply:
Yes, I want to describe the new way of configuring it and how to use it correctly.